src/libraries/System.Runtime.Serialization.Schema/src/System.Runtime.Serialization.Schema.csproj

```diff
@@ -0,0 +1,41 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <DefineConstants>$(DefineConstants)</DefineConstants>
+    <NoWarn>$(NoWarn);1634;1691;649</NoWarn>
+    <StrongNameKeyId>Microsoft</StrongNameKeyId>
+
+    <IsTrimmable>false</IsTrimmable>
+
+    <!-- NOTE TODO smolloy - This probably should be true... but this package needs to be added to some nuget feed first. dotnet build fails
+          if it looks for a package listing and doesn't find one... presumably because we say this should be
+          published as a package? -->
+    <IsPackable>false</IsPackable>
```

Comment:
Yes this should be true per https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/libraries-packaging.md

> dotnet build fails if it looks for a package listing and doesn't find one... presumably because we say this should be published as a package?

That doesn't sound right.  Can you share the error?  cc @ViktorHofer @dotnet/area-infrastructure-libraries

Reply:
My guess is that package baseline validation looks for the package on the feed and tries to restore it. You would need to disable PackageBaselineValidation in case this package doesn't exist yet on nuget with 6.0.0 version. You can achieve that by setting: 

```xml
<!-- TODO: Remove this setting when the package shipped with .NET 7. -->
<DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
```

Reply:
```suggestion
    <IsPackable>true</IsPackable>
    <!-- TODO: Remove this setting when the package shipped with .NET 7. -->
    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
```